igcollect/ping.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+"""igcollect - Latency collection
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+import _thread
+import argparse
+import subprocess
+
+from time import time, sleep
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Ping colecctor script for Graphana ')
+    parser.add_argument('adress', metavar='URL', type=str, nargs='+',
+                        help='the address zu ping to')
+    parser.add_argument('-c', '--count', type=int, default=20,
+                        help='how many tims should the script tries to ping')
+    parser.add_argument('--prefix', default='ping',
+                        help='the path to the value in Graphana')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    hostname = args.adress
+    count = args.count
+    prefix = args.prefix
+
+    i = 1
+    hosts = ''
+    for host in hostname:
+        hosts += host + ' '
+        if i % 10 == 0:
+            _thread.start_new_thread(ping, (prefix, hosts, count))
+            hosts = ''
+    if hosts != '':
+        ping(prefix, hosts, count)
+
+    while _thread._count() > 0:
+        sleep(1)
+
+
+def ping(prefix, hosts, count):
+    cmd = ('fping -B1 -q -C {} -p {} {}'.format(count, (20/count)*2000, hosts))
```

Comment:
two magic numbers here. what is `20` and what is `2000`? they should be constants at the very least, or even better optional arguments to the script

Reply:
this calculates the time between individual pings (itst the same than smokeping)